source/views/home/edit.js

```diff
@@ -200,18 +200,21 @@ class Row extends React.Component<void, RowProps, RowState> {
         <Text style={[styles.text, {color: this.props.data.tint}]}>
           {this.props.data.title}
         </Text>
-        {reorderIcon}
+        <Touchable highlight={false} {...this.props.sortHandlers}>
+          {reorderIcon}
```

Comment:
This at least makes just the reorder icon the only part of the cell that enables the drag... solving an issue we had on Android, I think.